silk/tools/otns_manager.py

```diff
@@ -130,6 +131,17 @@ def set_speed(self, speed: float):
     """
     self._send_command(f"speed {speed}")
 
+  def set_netinfo(self, version: str, commit: str):
+    """Set OTNS netinfo.
+
+    Args:
+      version (str): version string.
+      commit (str): commit string.
+    """
+    version_clause = f"version \"{version}\"" if version else ""
```

Comment:
```suggestion
    version_clause = f"version \"{version}\"" if version is not None else ""
```
We should allow set version to "". So we need to use `version is not None` check.